codemods/next/13/app-directory-boilerplate/test/test.ts

```diff
@@ -210,6 +210,136 @@ describe('next 13 app-directory-boilerplate', function () {
 		);
 	});
 
+	// Windows
+	// it.only('should build correct files', async function () {
```

Comment:
do you think we could run it in the CI/CD with a CLI flag? how much effort would that be?